packages/assets/scss/03-organisms/_callout-links-three-up.scss

```diff
@@ -0,0 +1,57 @@
+$bp-section-3up-large-min: "min-width:951px";
+$bp-section-3up-large-max: "max-width:950px";
+
+@use "00-base/configure" as *;
+
+.ma__callout-links-three-up {
+
+  &__title {
+
+    @include ma-container;
+  }
+
+
+  &__container {
+
+
+    @include clearfix;
+    display: flex;
+    align-items: stretch;
+    flex-wrap: wrap;
+    flex-direction: column;
+
+    @media ($bp-small-min) {
+      flex-flow: row wrap;
+    }
+
+    & > * {
+
+      @media ($bp-x-small-min) {
+        margin-bottom: 20px;
+      }
+
+      @media ($bp-small-min) {
+        flex-grow: 0;
+        flex-shrink: 0;
+        flex-basis: auto;
+        margin-bottom: 80px;
```

Comment:
<img width="1107" alt="Screen Shot 2022-05-05 at 4 50 42 PM" src="https://user-images.githubusercontent.com/5789411/167023440-d0315e7c-ae8e-431f-9421-652c406b003e.png">

this margin is not necessary
```suggestion
```